taskt/Core/Automation/Commands/If Commands/BeginMutliIfCommand.cs

```diff
@@ -177,15 +177,15 @@ private void IfConditionHelper_CellContentClick(object sender, DataGridViewCellE
 
                     var automationCommands = taskt.UI.CustomControls.CommandControls.GenerateCommandsandControls().Where(f => f.Command is BeginIfCommand).ToList();
                     frmCommandEditor editor = new frmCommandEditor(automationCommands, null);
-                    editor.selectedCommand = ifCommand;
-                    editor.editingCommand = ifCommand;
-                    editor.originalCommand = ifCommand;
-                    editor.creationMode = frmCommandEditor.CreationMode.Edit;
-                    editor.scriptVariables = ScriptVariables;
+                    editor.SelectedCommand = ifCommand;
+                    editor.EditingCommand = ifCommand;
+                    editor.OriginalCommand = ifCommand;
+                    editor.CreationModeInstance = frmCommandEditor.CreationMode.Edit;
+                    editor.ScriptVariables = ScriptVariables;
 
                     if (editor.ShowDialog() == DialogResult.OK)
                     {
-                        var editedCommand = editor.editingCommand as BeginIfCommand;
+                        var editedCommand = editor.EditingCommand as BeginIfCommand;
                         var displayText = editedCommand.GetDisplayValue();
                         var serializedData = Newtonsoft.Json.JsonConvert.SerializeObject(editedCommand);
```

Comment:
Could likely be reduced to JsonConvert.SerializeObject

Reply:
Same is the case here that it's just updating variable names/references in this Command File.